functions/api/routes/community/communityPostsGET.js

```diff
@@ -1,9 +1,68 @@
+const dayjs = require('dayjs');
+const customParseFormat = require('dayjs/plugin/customParseFormat');
+const util = require('../../../lib/util');
+const statusCode = require('../../../constants/statusCode');
+const responseMessage = require('../../../constants/responseMessage');
+const asyncWrapper = require('../../../lib/asyncWrapper');
+const db = require('../../../db/db');
+const { communityDB } = require('../../../db');
+const dummyImages = require('../../../constants/dummyImages');
+
 /**
  *  @route GET /community/posts
  *  @desc 커뮤니티 게시글 전체 조회
  *  @access Private
  */
 
-module.exports = async (req, res) => {
+module.exports = asyncWrapper(async (req, res) => {
   const { page, limit } = req.query;
-};
+  // page, limit이 없는 경우
+  if (!page || !limit) {
+    return res
+      .status(statusCode.BAD_REQUEST)
+      .send(util.fail(statusCode.BAD_REQUEST, responseMessage.NULL_VALUE));
+  }
+  // page, limit이 양수가 아닌 경우
+  if (isNaN(page) || isNaN(limit) || page < 1 || limit < 1) {
+    return res
+      .status(statusCode.BAD_REQUEST)
+      .send(util.fail(statusCode.BAD_REQUEST, responseMessage.OUT_OF_VALUE));
+  }
+
+  const dbConnection = await db.connect(req);
+  req.dbConnection = dbConnection;
+
+  dayjs().format();
+  dayjs.extend(customParseFormat);
+
+  const totalItemCount = parseInt(await communityDB.getCommunityPostsCount(dbConnection), 10); // 총 게시글 수
+  const totalPageCount = Math.ceil(totalItemCount / limit); // 총 페이지 수
+  const currentPage = parseInt(page, 10); // 현재 페이지
```

Comment:
```suggestion
  const currentPage = +page; // 현재 페이지
```

+ 연산자 사용하면 피연산자가 number가 아닐 시 변환해줘서 더 깔끔할 것 같습니다.

Reply:
parseInt를 씀으로써 좀 더 명시적으로 형변환을 해주려는 생각이었는데, 단순히 response에 쓰이는 데이터라서 타입변환을 명시적으로 보여줄 필요가 적은 것 같네요 😅
연산자를 이용하는게 더 깔끔한 것 같아용 수정하겠습니다 👍!